freqtrade/commands/strategy_utils_commands.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import os
+import time
+from typing import Any, Dict
+
+from freqtrade.configuration import setup_utils_configuration
+from freqtrade.enums import RunMode
+from freqtrade.resolvers import StrategyResolver
+from freqtrade.strategy.strategyupdater import StrategyUpdater
+
+
+logger = logging.getLogger(__name__)
+
+
+def start_strategy_update(args: Dict[str, Any]) -> None:
+    """
+    Start the strategy updating script
+    :param args: Cli args from Arguments()
+    :return: None
+    """
+
+    config = setup_utils_configuration(args, RunMode.UTIL_NO_EXCHANGE)
+
+    strategy_objs = StrategyResolver.search_all_objects(
+        config, enum_failed=False, recursive=config.get('recursive_strategy_search', False))
+
+    filtered_strategy_objs = []
+    if hasattr(args, "strategy_list"):
```

Comment:
there's something phishy going on with this

```
$ freqtrade strategy-updater --strategy-list TurtleStrategyX3p
2023-02-21 07:03:02,023 - freqtrade - INFO - freqtrade 2023.2.dev-08ca0f7c0
2023-02-21 07:03:02,023 - freqtrade.loggers - INFO - Verbosity set to 0
2023-02-21 07:03:02,024 - freqtrade.configuration.configuration - INFO - Using user-data directory: /home/xmatt/devel/cryptos/freqtrade/user_data ...
2023-02-21 07:03:02,024 - freqtrade.configuration.configuration - INFO - Using data directory: /home/xmatt/devel/cryptos/freqtrade/user_data/data ...
2023-02-21 07:03:02,024 - freqtrade.configuration.configuration - INFO - Using strategy list of 1 strategies
2023-02-21 07:03:02,024 - freqtrade.configuration.config_validation - INFO - Validating configuration ...
Conversion of sample_strategy.py started.
Conversion of sample_strategy.py took 0.0 seconds.
Conversion of TurtleStrategyX3p.py started.
Conversion of TurtleStrategyX3p.py took 0.0 seconds.
Conversion of StrategyB1.py started.
Conversion of StrategyB1.py took 0.0 seconds.
Conversion of xxyy.py started.
Conversion of xxyy.py took 0.0 seconds.
Conversion of xxyy1.py started.
Conversion of xxyy1.py took 0.0 seconds.
Conversion of StrategyB2.py started.
Conversion of StrategyB2.py took 0.0 seconds.

```

i've clearly passed in `--strategy-list` - but i got all strategies updated.